trieste/acquisition/optimizer.py

```diff
@@ -60,7 +65,7 @@ def _discrete_space(space: DiscreteSearchSpace, target_func: TensorMapping) -> T
 
 
 @optimize.register
-def _box(space: Box, target_func: TensorMapping) -> TensorType:
+def _box(space: Box, target_func: AcquisitionFunction) -> TensorType:
     trial_search_space = space.discretize(20 * tf.shape(space.lower)[-1])
     initial_point = optimize(trial_search_space, target_func)
```

Comment:
Victor and I were discussing this. It may be worth increasing the size of this initialisation (by quite a large amount), i.e. something that scales at a higher rate than linearly with input dimension.

Moreover, as we are using tensorflow, calculating the acquisition function across a much larger set of points is essentially the same cost as this smaller initialisation. In the past, I have used thousands of restarts for even smallish problems. 

This would be a great thing to test with our first COCO run?

Reply:
sounds good

Reply:
shall we do that in another PR?

Reply:
Yeah sure. I can do this. Maybe discuss in standup